src/Controller/CRUDController.php

```diff
@@ -59,22 +58,22 @@ class CRUDController extends Controller
      */
     private $templateRegistry;
 
-    public function setContainer(?ContainerInterface $container = null): void
+    public function setContainer(\Psr\Container\ContainerInterface $container): ?\Psr\Container\ContainerInterface
```

Comment:
I think better will be drop this and use inject service. Users will be able override this controller by own service too.

Reply:
I made something, tell me if it's what you wanted